qt/scientific_interfaces/Inelastic/Processor/DataProcessorInterface.cpp

```diff
@@ -51,7 +51,6 @@ std::string DataProcessorInterface::documentationPage() const { return "Inelasti
  */
 void DataProcessorInterface::exportTabPython() {
   auto const &tabName = m_uiForm.twIDRTabs->tabText(m_uiForm.twIDRTabs->currentIndex()).toStdString();
-  m_presenters[tabName]->exportPythonScript();
 }
```

Comment:
Does this functionality no longer work? There should be a button on the bottom of the tabs to "Export Python script"

Reply:
ahh.. thanks for catching this. It should work now.